apps/prairielearn/src/migrations/20230531184500_assessment_access_policies.sql

```diff
@@ -0,0 +1,25 @@
+CREATE SEQUENCE IF NOT EXISTS assessment_access_policies_id_seq;
+
+-- Create assessment_access_policies table
+CREATE TABLE IF NOT EXISTS assessment_access_policies (
+    id BIGINT NOT NULL DEFAULT nextval('assessment_access_policies_id_seq'::regclass),
```

Comment:
This is the conventional way to declare a primary key:

```suggestion
    id BIGSERIAL PRIMARY KEY,
```

You can remove the other explicit `CREATE SEQUENCE` and `PRIMARY KEY` bits.